Modules/Sources/Yosemite/Tools/Plugins/PluginsService.swift

```diff
@@ -16,20 +21,20 @@ public class PluginsService: PluginsServiceProtocol {
     }
 
     @MainActor
-    public func waitForPluginInStorage(siteID: Int64, pluginName: String, isActive: Bool) async -> SystemPlugin {
-        let predicate = \StorageSystemPlugin.siteID == siteID && \StorageSystemPlugin.name == pluginName && \StorageSystemPlugin.active == isActive
-        let nameDescriptor = NSSortDescriptor(keyPath: \StorageSystemPlugin.name, ascending: true)
+    public func waitForPluginInStorage(siteID: Int64, plugin: String, isActive: Bool) async -> SystemPlugin {
```

Comment:
Nit: How about updating the signature to `pluginPath`, or similar? Just to be more explicit about what we need to pass in.